### PR TITLE
Minor README updates and file additions

### PR DESCRIPTION
**Pull request by albinencio**
## Checklist

- [N/A] Did you write tests for your code?
- [N/A] Did all the tests pass locally?

## Trello Ticket

https://trello.com/c/QKww7c9H/87-update-readme-onboarding-doc
https://trello.com/c/mKPGOGsG/86-refactor-whole-documentation-to-be-self-contained

## Short Description

Made few changes to the README and added some files for documentation.

## Reviewer Notes

Anything special you want the reviewer to know?

### CODE REVIEW

**Review by Joyfal — APPROVED** [review]
LGTM

**Review by macedo22 — COMMENTED** [review]
Looks good! Left some remaining nitpicky grammar edit suggestions. Leaving others here in this comment since I don't think I can make inline review comments on lines that weren't changed:

line 61: "Be sure to checkout out the sections" -> "Be sure to check out the sections"
line 65: "A good reason to have two Firebase apps is to separate the production environment from the development environment, so that you have one more barrier of defense against having the production build potentially fail." -> remove the comma before "so that you have..."
 line 77: "Since we have two branches main and dev for production and development environments in our GitHub repository, we also created two apps on Firebase console and linked them to `prod` and `dev` config respectively." -> "As we have a `main` branch for production and a `dev` branch for developing, we also have a corresponding production and "beta" Firebase app, and we configure these two separately:"
 line 89: "you can remove `prod` and `dev` fields..." -> "you can remove the `prod` and `dev` fields..."

File: README.md [hunk 1]
```diff
@@ -4,32 +4,35 @@
...
+  - [Testing](#testing)
+    - [Jest](#jest)
+    - [Cypress](#cypress)
+    - [Code Coverage](#code-coverage)
+  - [Artifacts](#artifacts)
+  - [Maintenance](#maintenance)
 
 # [TaterTalk](https://twotenchatapp.web.app/)
 
 This is a chat web application that allows users to register/login with Google and provides 1 on 1 private chatting functionality.
```
[the remark this answers is not in the document]
Comment (albinencio) [reply README.md:32]: Done.

File: README.md [hunk 2]
```diff
@@ -4,32 +4,35 @@
...
 
+If you want to view our final presentation video, go [here](). You can access all final presentation materials in the [final_presentation](./final_presentation/) directory. If you want to read our retrospection blog, go [here](https://ucsd-mashed.github.io/).
+
 ## Architecture
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app) and utilizes [Firebase](https://firebase.google.com/), more specifically, it relies on Firebase Authentication, Firestore and Hosting.
```
[the remark this answers is not in the document]
Comment (albinencio) [reply README.md:38]: Done.

File: README.md [hunk 3]
```diff
@@ -58,11 +58,11 @@ Welcome aboard! Below are the steps you need to take to get up and running, as w
 5. Run `yarn install` to install our project's dependencies.
 6. Start the local server by doing `yarn start` and explore!
 
-You're all set! You can visit the [Scripts](#Scripts) section to learn more about other useful commands if you are interested. Be sure to checkout out the sections below, where you can find out more about our design, specifications, documentation, and testing. If you are joining our team, this is also where you will find our [Contributing Guidelines](./artifacts/ContributingGuidelines.pdf).
+You're all set! You can visit the [Scripts](#Scripts) section to learn more about other useful commands if you are interested. Be sure to check out out the sections below, where you can find out more about our design, specifications, documentation, and testing. If you are joining our team, this is also where you will find our [Contributing Guidelines](./artifacts/ContributingGuidelines.pdf).
```
Comment (macedo22) [inline README.md:61]: Oops I think this one was me not being clear in what I meant - we have "out" repeated "check out out" -> "check out"

**Review by macedo22 — CHANGES_REQUESTED** [review]
I am so sorry

**Review by macedo22 — APPROVED** [review]

**Pull request merged**